src/components/loader/loader.tsx

```diff
@@ -20,9 +20,21 @@ export class Loader {
   //  Properties
   //
   //--------------------------------------------------------------------------
-  /** When `true`, the component is active. */
+
+  /**
+   * When `true`, the component is active.
+   *
+   * @deprecated use global `hidden` attribute instead.
+   */
   @Prop({ reflect: true }) active = false;
 
+  /**
+   * When `true`, the component will not be visible.
+   *
+   * @mdn [hidden](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden)
```

Comment:
we shouldn't need to document or add this property since its already a global property on every dom element.